bin/critterget.py

```diff
@@ -555,7 +555,12 @@ def getDailyCrashes(appId,appName):
 
 
 def getTrends(appId,appName):
-    """Get the number of daily app crashes for all versions of an app."""
+    """Calls the trends endpoint for an app
+
+    :param appId: (string) App ID used to request data from the API
```

Comment:
These look like intelij docstrings.  That's fine, but they look a little ugly to me.  The Google Python Style Guide is what I've used for more readable docstrings: https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments

Up to you, just wanted to share what I use.